one-pager-maker/src/Header.tsx

```diff
@@ -0,0 +1,31 @@
+import {signOut} from 'firebase/auth';
+import {auth} from './firebase.ts';
+import {Link} from 'react-router-dom';
+import {useAppSelector} from './redux/hooks';
+import {selectUser} from './redux/user/selector.ts';
+import {Button} from '@mui/material';
+import OnePagerMakerIcon from "./assets/OnePagerMaker.png"
+
+function Header() {
+    const user = useAppSelector(selectUser);
+    const handleSignOut = () => signOut(auth).catch(error =>console.error('Sign out failed:', error));
+    return (
+        user ? (
+        <header>
+            <nav className="relative px-4 py-4 flex justify-between items-center bg-white">
```

Comment:
You might not need nav here?

Instead, perhaps it is better to use nav in line 20:
```tsx
<nav>
                <ul className="absolute top-1/2 left-1/2 transform -translate-y-1/2 -translate-x-1/2 flex mx-auto items-center w-auto space-x-6">
                    <li><Link className="text-sm text-gray-400 hover:text-gray-500" to="/">Home</Link></li>
                    <li>|</li>
                    <li><Link className="text-sm text-gray-400 hover:text-gray-500" to="/">About</Link></li>
                </ul>
</nav>
```

Reply:
fixed